gateway-ha/src/test/java/io/trino/gateway/CustomTrinoImageNameSubstitutor.java

```diff
@@ -25,7 +25,7 @@ public class CustomTrinoImageNameSubstitutor
         extends ImageNameSubstitutor
 {
     private static final String TESTCONTAINERS_TRINO_IMAGE_SUBSTITUTE = "TESTCONTAINERS_TRINO_IMAGE_SUBSTITUTE";
-    private static final String TRINO_IMAGE = "trinodb/trino";
+    private static final String TRINO_IMAGE = "trinodb/trino:476";
```

Comment:
this is the wrong file. This file is to substitute the `trinodb/trino` image. We should change the version in https://github.com/trinodb/trino-gateway/blob/92e927401d786ccdc1cd72213c7d928ce1247c4e/gateway-ha/src/test/java/io/trino/gateway/ha/clustermonitor/TestClusterStatsMonitor.java#L50